src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ReadMetadata.java

```diff
@@ -361,6 +361,15 @@ public static void writeMetadata(final ReadMetadata readMetadata,
                                 "+" + stats.getPositiveMAD() + "\t" + stats.getCoverage() + "\t" +
                                 stats.getMeanBaseQuality() + "\t" +
                                 stats.getNReads() + "\t" + stats.getReadStartFrequency() + "\n");
+                final IntHistogram.CDF templateSizeCdf = stats.getCDF();
+                final int cdfSize = templateSizeCdf.size();
+                final long totalObservations = templateSizeCdf.getTotalObservations();
+                writer.write("template size cumulative counts:");
```

Comment:
Was going to suggest capitalizing to be consistent with "Library Statistics" but then realized that the headings are all over the place in this file in terms of formatting. Don't feel like you need to do it for this pull request but it would be nice to eventually convert the metadata into a more easily machine-parseable format eventually (JSON?).